homework/PavelBo/dataTypes/encoder/src/main/java/com/github/pavelbopavel/Encoder.java

```diff
@@ -0,0 +1,19 @@
+package com.github.pavelbopavel;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class Encoder {
+
+    Logger log = LoggerFactory.getLogger(Encoder.class);
+
+    public int decoder(char symbol){
+        log.info("code = {}",(int) symbol);
+        return (int) symbol;
```

Comment:
it better to use variable to store result of `(int) symbol` calculation  and than print it and return it instead of calculating it twice.